agent/flux.go

```diff
@@ -28,7 +112,7 @@ func getFluxConfig(k kubectl.Client, namespace string) (*FluxConfig, error) {
 		return nil, err
 	}
 
-	if cfg.GitBranch == "" && cfg.GitLabel == "" && cfg.GitPath == "" && cfg.GitURL == "" {
+	if cfg.AsQueryParams() == "" {
```

Comment:
A better way to do this @squaremo ?

Reply:
If nothing is set, it should be equal to the zero value `FluxConfig{}` (which will be all nil pointers).